docsrc/source/basics.rst

```diff
@@ -87,10 +87,9 @@ The returned output is
 
 .. code-block:: python
 
-    pathways =[[0.7,  NaN],
-               [0.3,   0 ]]
+    pathways =[[0.7], [0.3,   0 ]]
```

Comment:
Make it possible to use `[0.7, [0,3.0]]`.

Reply:
It is already possible. I took care of that.